binder/postBuild

```diff
@@ -37,3 +37,15 @@ jupyter labextension link --no-build packages/application/
 jupyter labextension link --no-build packages/ui-components/
 jupyter lab build --dev-build=False --minimize=False
 mkdir -p binder-demo
+
+set -x
```

Comment:
I've added this in mainly for troubleshooting. Technically this can be removed but it doesn't hurt either.